### PR TITLE
Bootstrap5

### PR DESCRIPTION
**Pull request by Mryam1**
**Updated Templates to bootstrap5**

Summary of the changes (Less than 80 chars)

- Updated the web templates to use the latest version of bootstrap.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=33592) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=33592) before we can accept your contribution.<br/><br/>:x: Mryam1 [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=33592)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=33592) it.</sub>

File: src/Identity/UI/src/Areas/Identity/Pages/V5/Account/AccessDenied.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Microsoft.AspNetCore.Identity.UI.V4.Pages.Account.Internal
```
Comment (HaoK) [inline src/Identity/UI/src/Areas/Identity/Pages/V5/Account/AccessDenied.cshtml.cs:6]: I think we might want to change these to be V5 namespaces right @javiercn or will this change so it doesn't matter?

**Comment by SteveSandersonMS** [comment]
Looking forwards to seeing this in action, @Mryam1!

One tricky thing about our project template infrastructure is that it's very difficult to get a runnable project from the source code, which complicates the review. Do you have some way of executing these updated templates to get an example of the output project? I totally understand if not, but if it's possible to make a small repo somewhere containing examples of the emitted projects (that are runnable) that would be extremely beneficial.

**Comment by LadyNaggaga** [comment]
> Do you have some way of executing these updated templates to get an example of the output project? 
@SteveSandersonMS Mryam test each template locally with the template with the [project template scripts](https://github.com/dotnet/aspnetcore/tree/main/src/ProjectTemplates/scripts) and will respond to this PR with a few screenshots.  She noticed that she couldn't test the Identity UI locally. We tried passing the -au Individual in the scripts, and we get a bunch of errors.

**Comment by pranavkm** [comment]
Could @Mryam1 publish the generated projects from scripts directory to a GitHub repo? Might be quicker than having to build the project.

**Comment by Mryam1** [comment]
Here is a screenshot of the webapp template running locally with bootstrap5.

image


> Could @Mryam1 publish the generated projects from scripts directory to a GitHub repo? Might be quicker than having to build the project.

What are the steps to doing this? @pranavkm

**Comment by HaoK** [comment]
Merged this branch and continuing this PR in https://github.com/dotnet/aspnetcore/pull/34052 still working through the issues around identity UI, but I was able to generate current outputs of the templates which don't use auth which can be reviewed already:

https://github.com/HaoK/Random/tree/boot5/Templates

**Comment by HaoK** [comment]
Replaced by https://github.com/dotnet/aspnetcore/pull/34052

**Pull request closed without merging**